meta-balena-common/recipes-support/development-features/development-features/development-features

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+DEVELOPMENT_FEATURES_FLAG_FILE="/var/volatile/development-features"
+
+. /usr/sbin/balena-config-vars
+
+if [ "${DEVELOPMENT_IMAGE}" = "true" ] && [ ! -f "${DEVELOPMENT_FEATURES_FLAG_FILE}" ]; then
+	echo "SSHD_OPTS='-f /etc/ssh/sshd_config_development'" > "${DEVELOPMENT_FEATURES_FLAG_FILE}"
+	echo 'BALENA_DEVELOPMENT_ARGS="-H tcp://0.0.0.0:2375"' >> "${DEVELOPMENT_FEATURES_FLAG_FILE}"
+	echo "[INFO] Entering development mode"
+	/bin/systemctl restart development-features.target
+else
+	if [ -f "${DEVELOPMENT_FEATURES_FLAG_FILE}" ]; then
+		rm -rf "${DEVELOPMENT_FEATURES_FLAG_FILE}"
+		echo "[INFO] Entering production mode"
```

Comment:
You could include `/usr/libexec/os-helpers-logging` and use `info "message..."` for the 2 lines above instead of `echo "[INFO] message..."`

Reply:
Totally, thanks for the reminder @markcorbinuk